README.md

```diff
@@ -167,6 +167,56 @@ module.exports = {
   },
 };
 ```
+## Metrics Middleware
+In supports exporting custom application metrics that can be picked by *Prometheus*.
+In order to use this middleware you should also enable the *prometheus plugin* described in the following section.
+
+Example route configuration
+
+```js
+const { middlewares: { metrics } } = require('@workablehr/orka');
+
+module.exports = {
+  get: {
+    '/metrics': metrics,
+    '/users': async (ctx, next) => {
+      ctx.body = await User.find();
+    }
+  }
+};
+```
+## Prometheus metrics
+Prometheus is optional. In order to enable it install the dependency first:
+
+```sh
+npm i prom-client
+```
+And add to your configuration
+
+```js
+{
+  prometheus:{
+    enabled: true
+  }
+}
+```
+
+I can either be used in conjunction with the `metrics` middleware (pull mode) or if you wish to use it in a non web context, using the *PushGateway* via the provided `#push()` method.
```

Comment:
```suggestion
It can either be used in conjunction with the `metrics` middleware (pull mode) or if you wish to use it in a non web context, using the *PushGateway* via the provided `#push()` method.
```